LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Smartcar, Inc. <hello@smartcar.com>
+Copyright (c) 2020 Smartcar, Inc. <hello@smartcar.com>
```

Comment:
nit should we do `2014-2020` ?

Reply:
looked around at a few examples on this, I think we should be fine with just `2020`